pkg/connections/stomp/connection_test.go

```diff
@@ -32,6 +34,14 @@ import (
 // Utility functions used for testing.
 //
 
+// DestinationName generate a runadom destination name
```

Comment:
s/runadom/random/

If I understand correctly what you want is to ensure that you have an empty queue before starting the test. Using a random name may be OK, but it will leave a lot of queues around, and that may be problematic. Instead of that I'd suggest to create two methods: one to create a queue and another to remove it. Then each test should make sure that both are called:

```go
func MyTest(...) {
    queue, err := CreateTestQueue()
    if err != nil {
        ....
    }
    defer DeleteTestQueue(queue)

    ...
}
```

Your initial implementation of `CreateTestQueue` can be the same that you do now in `DestinationName`, and `DeleteTestQueue` can be empty. Later you can change those implementations so that they actually create/remove the queues, using the `artemis queue create` and `artemis queue delete` commands, for example.
}

Reply:
Thanks, fixed the typo !

p.s.
`artemis queue delete` is not what I was looking for .. I can just close the artimis, and start it again ... , what I need is a STOMP command [1] to remove queue.

[1] https://stomp.github.io/stomp-specification-1.2.html#STOMP_Frames